extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -66,6 +66,11 @@ class AmpAccordion extends AMP.BaseElement {
 
     /** @private {number|string} */
     this.prefix_ = element.id ? element.id : Math.floor(Math.random() * 100);
+
+    /** @const @private {!MutationObserver} */
+    this.mutationObserver_ = new this.win.MutationObserver(mutations => {
```

Comment:
There is no need to hold a reference to this. Would be cleaner to just have this created in the `buildCallback`.